### ISSUE TITLE
Docker image failing on openshift

### ISSUE THREAD

**Issue by im-pratham** [flowable/flowable-engine#2941]
**Describe the bug**
Docker image flowable/flowable-ui is failing to run on openshift cluster due to following error - 
`
./wait-for-something.sh: exec: line 16: /flowable-entrypoint.sh: Permission denied
`

**Expected behavior**
Image should run on openshift

**Code**

**Additional context**
The Openshift platform runs each pod with randomly selected user, so we account this while creating the docker image. See https://docs.openshift.com/container-platform/4.3/openshift_images/create-images.html for more info. 
The error "Permission denied" occurs because the file is owned by root and file permission we set is 544.
![image](https://user-images.githubusercontent.com/22445437/120477006-df2bd780-c3c8-11eb-959e-80b462649afb.png)

*The openshift user is part of root group, so we need to allow group to execute file.*

In addition to that, su-exec will fail to execute on openshift with error
`
su-exec: setgroups: Operation not permitted
`
So we need to take same thing in consideration. 

I have created the PR - https://github.com/flowable/flowable-engine/pull/2940
The image created using PR can be used and tested from here - https://hub.docker.com/r/impratham/flowable-ui

**Comment by yvoswillens** [comment]
Hi,

could you provide a more complete stack trace?
We already applied changes that result in running the 'Flowable application process' as non root. Also with Openshift in mind.

```
PID   USER     TIME  COMMAND
    1 flowable  0:42 java -Xmx1024M -Djava.awt.headless=true -Dfile.encoding=UTF-8 -cp /app:/app/WEB-INF/classes:/app/WEB-INF/lib/* org.flowable.ui.application.FlowableUiApplication
   65 root      0:00 ash
   72 root      0:00 ps
```

**Comment by im-pratham** [comment]
Hi @yvoswillens, it fails for after wait_for_something finishes.
Here is console output 
```bash
PostgreSQL is up - executing command
./wait-for-something.sh: exec: line 16: /flowable-entrypoint.sh: Permission denied
```
You can check file permissions on container attached above

**Comment by im-pratham** [comment]
resolves #2943
